test/Microsoft.Azure.Devices.Edge.Test.Common/certs/RootCertificateAuthority.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Certs
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public class RootCertificateAuthority
+    {
+        readonly string scriptPath;
+
+        public static async Task<RootCertificateAuthority> CreateAsync(string certificatePath, string keyPath, string password, string scriptPath, CancellationToken token)
+        {
+            await Platform.InstallRootCertificateAsync(certificatePath, keyPath, password, scriptPath, token);
+            return new RootCertificateAuthority(
+                certificatePath,
+                keyPath,
+                scriptPath);
+        }
+
+        RootCertificateAuthority(string certificatePath, string keyPath, string scriptPath)
```

Comment:
why are certificatePath and keyPath not used?

Reply:
Oh yeah, forgot to remove those--I'll do it. These classes wrap the cert scripts under `tools/CACertificates`, which assume that certain certificates live in certain locations. So you don't need to pass paths into them.

Reply:
Done.